packages/flutter_tools/lib/src/ios/migrations/remove_bitcode_migration.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../base/file_system.dart';
+import '../../base/project_migrator.dart';
+import '../../xcode_project.dart';
+
+/// Remove deprecated bitcode build setting.
+class RemoveBitcodeMigration extends ProjectMigrator {
+  RemoveBitcodeMigration(
+    IosProject project,
+    super.logger,
+  )   : _xcodeProjectInfoFile = project.xcodeProjectInfoFile;
+
+  final File _xcodeProjectInfoFile;
+
+  @override
+  bool migrate() {
+    if (_xcodeProjectInfoFile.existsSync()) {
+      processFileLines(_xcodeProjectInfoFile);
+    } else {
+      logger.printTrace('Xcode project not found, skipping removing bitcode migration.');
+    }
+
+    return true;
+  }
+
+  @override
+  String? migrateLine(String line) {
+    if (line.contains('ENABLE_BITCODE = YES;')) {
+      if (!migrationRequired) {
+        // Only print for the first discovered change found.
+        logger.printWarning('Disabling deprecated bitcode Xcode build setting. See https://github.com/flutter/flutter/issues/107887 for additional details.');
+      }
+      return line.replaceAll('ENABLE_BITCODE = YES', 'ENABLE_BITCODE = NO');
```

Comment:
is the spacing here guaranteed to be a single space on both sides of the `=`?

Reply:
I mean it's totally undocumented but it's been that way for as long as I've been mucking with Xcode projects.